Makefile.oss.prow

```diff
@@ -52,7 +32,7 @@ build-gke-e2e:
 		.
 
 .PHONY: push-gke-e2e
-push-gke-e2e:
+push-gke-e2e: build-gke-e2e
```

Comment:
Personally I prefer the flexibility of being able to build/push separately, so that I can sanity check before pushing

Reply:
I have no strong preference. Reverted this part.